applications/sandbox/systems/gui_test.hpp

```diff
@@ -3,13 +3,14 @@
 #include <rendering/rendering.hpp>
 #include <rendering/util/gui.hpp>
 #include <rendering/pipeline/gui/stages/imguirenderstage.hpp>
+#include <rendering/systems/renderer.hpp>
 
-using namespace legion::rendering;
+using namespace rendering;
```

Comment:
there's no such namespace as ::rendering why change this from legion::rendering?

Reply:
it's a tooling issue, but this does in fact compile

Reply:
I'll revert that stuff tomorrow